jans-cedarling/schema/cedarling_core.cedarschema

```diff
@@ -90,42 +91,42 @@ namespace Jans {
 
     // ******  Actions  ******
     action Compare appliesTo {
-      principal: [User, Workload, Role],
+      principal: [User, Workload],
```

Comment:
Why did you remove role for all these?

Reply:
The role was added as a workaround because Agama lab could not create RBAC policies without them being Role defined in actions, even if this was legal in cedar. This bug has since been fixed.